libs/nx-vite/src/executors/serve/executor.ts

```diff
@@ -13,7 +13,7 @@ export default async function runExecutor(
     const tree = new FsTree(context.cwd, context.isVerbose)
     const appRoot = context.workspace.projects[context.projectName].root
 
-    const vite = execa('vite', [`${tree.root}/${appRoot}`, '--open'], {
+    const vite = execa('yarn vite', [`${tree.root}/${appRoot}`, '--open'], {
```

Comment:
Could you use https://github.com/sevenwestmedia-labs/nx-plugins/blob/main/libs/nx-esbuild/src/executors/package/executor.ts#L17 and detect the package manager, then `yarn` or `pnpx` depending on the detected package manager?

Reply:
yip